.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/node:8.11.3
+      - image: circleci/node:14
```

Comment:
Generally with our libraries we use the oldest supported LTS version (or even better, [run tests on _all_ supported LTS versions](https://github.com/MetaMask/eth-json-rpc-filters/pull/45)).

I'd suggest changing this to v10 (the current oldest supported LTS version). Or maybe v12, since v10 is EOL at the end of next month.

Reply:
I did check earlier that tests pass using both v10 and v12. Only v8 fails.